include/boost/math/distributions/logistic.hpp

```diff
@@ -145,12 +147,8 @@ namespace boost { namespace math {
           return result;
        }
        BOOST_MATH_STD_USING
-       RealType power = (location - x) / scale;
-       if(power > tools::log_max_value<RealType>())
-          return 0;
-       if(power < -tools::log_max_value<RealType>())
-          return 1;
-       return 1 / (1 + exp(power)); 
+       RealType power = -(location - x) / scale;
```

Comment:
Very minor nitpick: The extra `-` seems unnecessary. `(x - location)/scale` would be usual translation from the `(location, scale)` family to the standard value.